Misc/NEWS.d/next/Library/2019-08-27-03-57-25.bpo-37958.lRORI3.rst

```diff
@@ -0,0 +1,7 @@
+pstats is really useful or profiling and printing the output of the execution of some block of code, but I've found on multiple occasions when I'd like to access this output directly in an easily usable dictionary on which I can further analyze or manipulate.

```

Comment:
A news entry should just state what is new, not be text selling it or proposing it.

Something mundane such as "Added the pstats.Stats.get_profile_dict() method to get the profile data as a dictionary."

Reply:
Done.